Tests/PINRemoteImageTests.m

```diff
@@ -943,6 +944,51 @@ - (void)testQOS
     XCTAssert(dispatch_semaphore_wait(semaphore, [self timeout]) == 0, @"Semaphore timed out.");
 }
 
+- (void)testMaxAge
```

Comment:
Thanks for adding tests! Maybe call this testCacheControlMaxAgeExpiry, or just testCacheControl? Since the maxAge comes from cache control, the cache-control seems like the root nature / focus of the test.

Reply:
"testCacheControlSupport"?